### DISCUSSION TITLE
blank image after job done for trying new cartoon style

### DISCUSSION THREAD

**Discussion by lgwjames** [lllyasviel/Fooocus#3069] [Q&A] [answered]
Hi all, 

No image after job finished? and yes, nothing in output folder but 2 black images

![screenshot-127 0 0 1_7865-2024 06 01-09_03_30](https://github.com/lllyasviel/Fooocus/assets/35565479/481cb090-1774-427e-8774-e60aec7e9736)

Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 5.89 seconds
huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
	- Avoid using `tokenizers` before the fork if possible
	- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
  0%|                                                    | 0/60 [00:00<?, ?it/s]/Users/Quad-Core/Fooocus/modules/anisotropic.py:132: UserWarning: The operator 'aten::std_mean.correction' is not currently supported on the MPS backend and will fall back to run on the CPU. This may have performance implications. (Triggered internally at /Users/runner/work/pytorch/pytorch/pytorch/aten/src/ATen/mps/MPSFallback.mm:13.)
  s, m = torch.std_mean(g, dim=(1, 2, 3), keepdim=True)
 75%|████████████████████████████████▎          | 45/60 [12:40<04:13, 16.89s/it]Requested to load SDXLRefiner
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 1.65 seconds
Refiner Swapped
100%|███████████████████████████████████████████| 60/60 [16:13<00:00, 16.23s/it]
Requested to load AutoencoderKL
Loading 1 new model
/Users/Quad-Core/Fooocus/modules/core.py:331: RuntimeWarning: invalid value encountered in cast
  return [np.clip(255. * y.cpu().numpy(), 0, 255).astype(np.uint8) for y in x]
[Fooocus] Saving image 1/4 to system ...
Image generated with private log at: /Volumes/1TB Caddy bay 5/2024-06-01/log.html
Generating and saving time: 984.76 seconds
[Fooocus] Preparing task 2/4 ...
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
 75%|████████████████████████████████▎          | 45/60 [12:17<03:58, 15.88s/it]Refiner Swapped
100%|███████████████████████████████████████████| 60/60 [15:34<00:00, 15.58s/it]
[Fooocus] Saving image 2/4 to system ...
Image generated with private log at: /Volumes/1TB Caddy bay 5/2024-06-01/log.html
Generating and saving time: 938.55 seconds
[Fooocus] Preparing task 3/4 ...
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
 55%|█████████████████████████████████████████████████████████████████████████████████████

**Comment by mashb1t** [comment]
> Loading 1 new model
/Users/Quad-Core/Fooocus/modules/core.py:331: RuntimeWarning: invalid value encountered in cast
return [np.clip(255. * y.cpu().numpy(), 0, 255).astype(np.uint8) for y in x]

Can you please provide the data for this image from the history log (copy to clipboard)?

**Comment by lgwjames** [comment]
here you go,  setting sai ( was told to use this one ) 

{
"prompt": "A English rose woman, blonde hair, (blue eyes), light freckles around nose and cheeks. 32dd breasts, perfect eyes, perfect hands, sitting in the cafe, drinking a cup of tea, comic, graphic illustration, comic art, graphic novel art, vibrant, highly detailed, colored, 2d minimalistic",
"negative_prompt": "_neg_",
"prompt_expansion": "A English rose woman, blonde hair, (blue eyes), light freckles around nose and cheeks. 32dd breasts, perfect eyes, perfect hands, sitting in the cafe, drinking a cup of tea, comic, graphic illustration, comic art, graphic novel art, vibrant, highly detailed, colored, 2d minimalistic, epic, cinematic, professional, artistic, sharp",
"styles": "['Fooocus V2', 'Fooocus Cinematic']",
"performance": "Quality",
"resolution": "(1152, 896)",
"guidance_scale": 7,
"sharpness": 2,
"adm_guidance": "(1.5, 0.8, 0.3)",
"base_model": "sd_xl_base_1.0_0.9vae.safetensors",
"refiner_model": "sd_xl_refiner_1.0_0.9vae.safetensors",
"refiner_switch": 0.75,
"clip_skip": 2,
"sampler": "dpmpp_2m_sde_gpu",
"scheduler": "karras",
"vae": "Default (model)",
"seed": "2363294541890250693",
"lora_combined_1": "sd_xl_offset_example-lora_1.0.safetensors : 0.5",
"lora_combined_2": "JessicaL.safetensors : 1.0",
"lora_combined_3": "Perfect Hands v2.safetensors : 1.0",
"lora_combined_4": "PerfectEyesXL.safetensors : 1.0",
"metadata_scheme": false,
"version": "Fooocus v2.4.1"
}

**Comment by lgwjames** [comment]
its seems the sai setting just outputs blanks

**Comment by lgwjames** [comment]
new error when i add face swap, normal set up , it made a image, added a image of some friends and next job black screen

[Prompt Expansion] boy meets girl, Cutest cartoon illustration, empty background, cute, colorful, beautiful, artstation, deviantart, pinterest, highly detailed, sharp focus, intricate, elegant, confident, vibrant, color, inspired, pretty, friendly, innocent, epic, joyful, perfect, full, focused, loving, professional, unique, amazing, creative, best, gorgeous
[Fooocus] Encoding positive #1 ...
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding positive #3 ...
[Fooocus] Encoding positive #4 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Fooocus] Encoding negative #3 ...
[Fooocus] Encoding negative #4 ...
[Fooocus] Image processing ...
Downloading: "https://github.com/xinntao/facexlib/releases/download/v0.1.0/detection_Resnet50_Final.pth" to /Users/Quad-Core/Fooocus/models/controlnet/detection_Resnet50_Final.pth

100%|████████████████████████████████████████████████████████████████████████████████████████| 104M/104M [00:12<00:00, 8.66MB/s]
Downloading: "https://github.com/xinntao/facexlib/releases/download/v0.2.2/parsing_parsenet.pth" to /Users/Quad-Core/Fooocus/models/controlnet/parsing_parsenet.pth

100%|██████████████████████████████████████████████████████████████████████████████████████| 81.4M/81.4M [00:09<00:00, 8.65MB/s]
Detected 2 faces
Requested to load CLIPVisionModelWithProjection
Loading 1 new model
Requested to load Resampler
Loading 1 new model
Requested to load To_KV
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.14 seconds
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (896, 1152)
Preparation time: 179.78 seconds
[Fooocus] Preparing task 1/4 ...
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
unload clone 5
[Fooocus Model Management] Moving model(s) has taken 5.02 seconds
  0%|                                                                                                    | 0/60 [00:00<?, ?it/s]/Users/Quad-Core/Fooocus/modules/core.py:257: RuntimeWarning: invalid value encountered in cast
  x_sample = x_sample.cpu().numpy().clip(0, 255).astype(np.uint8)
100%|███████████████████████████████████████████████████████████████████████████████████████████| 60/60 [15:21<00:00, 15.35s/it]
/Users/Quad-Core/Fooocus/modules/core.py:331: RuntimeWarning: invalid value encountered in cast

**Comment by lgwjames** [comment]
{
"prompt": "boy meets girl, Cutest cartoon illustration, empty background, cute, colorful, beautiful, artstation, deviantart, pinterest",
"negative_prompt": "jumbled",
"prompt_expansion": "boy meets girl, Cutest cartoon illustration, empty background, cute, colorful, beautiful, artstation, deviantart, pinterest, highly detailed, cinematic, intricate, sharp focus, inspired, innocent, fine color, dramatic light, elegant, sublime, vibrant colors, epic, excellent composition, dynamic, atmosphere, full, crisp, very inspirational, ambient, lively",
"styles": "['Fooocus V2', 'Fooocus Enhance', 'Fooocus Sharp']",
"performance": "Quality",
"resolution": "(1152, 896)",
"guidance_scale": 4,
"sharpness": 2,
"adm_guidance": "(1.5, 0.8, 0.3)",
"base_model": "juggernautXL_v8Rundiffusion.safetensors",
"refiner_model": "None",
"refiner_switch": 0.5,
"clip_skip": 2,
"sampler": "dpmpp_2m_sde_gpu",
"scheduler": "karras",
"vae": "Default (model)",
"seed": "3902016816918476853",
"lora_combined_1": "PerfectEyesXL.safetensors : 1.0",
"lora_combined_2": "Perfect Hands v2.safetensors : 1.0",
"metadata_scheme": false,
"version": "Fooocus v2.4.1"
}

**Comment by lgwjames** [comment]
but this one worked

{
"prompt": "boy meets girl, Cutest cartoon illustration, empty background, cute, colorful, beautiful, artstation, deviantart, pinterest",
"negative_prompt": "low resolution",
"prompt_expansion": "boy meets girl, Cutest cartoon illustration, empty background, cute, colorful, beautiful, artstation, deviantart, pinterest, highly detailed, dramatic light, cinematic, symmetry, clear focus, aesthetic, shiny, great composition, colors, perfect, epic, fine detail, stunning, gorgeous, full color, intricate, elegant, amazing, creative, positive radiant",
"styles": "['Fooocus V2', 'Fooocus Enhance', 'Fooocus Sharp']",
"performance": "Quality",
"resolution": "(1152, 896)",
"guidance_scale": 4,
"sharpness": 2,
"adm_guidance": "(1.5, 0.8, 0.3)",
"base_model": "juggernautXL_v8Rundiffusion.safetensors",
"refiner_model": "None",
"refiner_switch": 0.5,
"clip_skip": 2,
"sampler": "dpmpp_2m_sde_gpu",
"scheduler": "karras",
"vae": "Default (model)",
"seed": "5292573490327118572",
"lora_combined_1": "PerfectEyesXL.safetensors : 1.0",
"lora_combined_2": "Perfect Hands v2.safetensors : 1.0",
"metadata_scheme": false,
"version": "Fooocus v2.4.1"
}
![2024-06-01_11-04-36_4167](https://github.com/lllyasviel/Fooocus/assets/35565479/17667eed-340e-4927-9b7d-dceb882ce4de)

**Comment by mashb1t** [comment, marked as answer]
see https://github.com/lllyasviel/Fooocus/issues/3070#issuecomment-2143818833
